### PR TITLE
Scheduled retries aggregated to rate per second

### PR DESCRIPTION
**Pull request by tmasternak**
## Overview
This PR changes how Scheduled Retries are aggregated. It uses the same approach as Throughput i.e. calculates rate or measurements per second.

Tests for retry aggregations have been removed as aggregation is done with `AggregateTotalMeasurementsPerSecond` which is already covered.

### CODE REVIEW

File: src/ServiceControl.Monitoring/Http/Diagrams/DiagramApiModule.cs [hunk 1]
```diff
@@ -28,7 +28,7 @@ public MonitoredEndpointsModule(IProvideBreakdownBy<EndpointInstanceId>[] metric
             {
                 CreateMetric<EndpointInstanceId, ProcessingTimeStore>("ProcessingTime", IntervalsAggregator.AggregateTimings),
                 CreateMetric<EndpointInstanceId, CriticalTimeStore>("CriticalTime", IntervalsAggregator.AggregateTimings),
-                CreateMetric<EndpointInstanceId, RetriesStore>("Retries", IntervalsAggregator.AggregateRetries),
```
Comment (mikeminutillo) [inline src/ServiceControl.Monitoring/Http/Diagrams/DiagramApiModule.cs]: Can `IntervalsAggregator.AggregateRetries` also be 🔥 ?

Comment (tmasternak) [reply src/ServiceControl.Monitoring/Http/Diagrams/DiagramApiModule.cs]: done.

**Comment by tmasternak** [comment]
@Scooletz note on test removal added in the description.

**Comment by tmasternak** [comment]
@mikeminutillo review changes are in place

**Pull request merged**